config/src/config/network_config.rs

```diff
@@ -29,8 +29,8 @@ use std::{
 // might need to extract into a separate network_constants crate or something.
 pub const HANDSHAKE_VERSION: u8 = 0;
 
-pub type NetworkPeersConfig = HashMap<PeerId, x25519::PublicKey>;
-pub type SeedPeersConfig = HashMap<PeerId, Vec<NetworkAddress>>;
+pub type SeedPubkeysConfig = HashMap<PeerId, HashSet<x25519::PublicKey>>;
```

Comment:
the term "config" seems redundant for this and below... it is just SeedPub(lic)Keys and SeedAddr(esse)s

Reply:
agreed  : )